package.json

```diff
@@ -52,6 +52,6 @@
   "dependencies": {
     "diagnostic-channel": "0.2.0",
     "diagnostic-channel-publishers": "0.2.1",
-    "zone.js": "0.7.6"
+    "zone.js": "^0.8.17"
```

Comment:
I would be wary of updating `zone.js` without other changes to the `AppInsightsAsyncCorrelatedErrorWrapper`. Last time I tried this, various versions of node gave errors related to restricted properties. Once the type error is fixed to allow the unit tests to continue, I would expect you'll get the same errors reported in unit tests for quite a few node versions.

We also restrict our dependencies to a specific version for support reasons, so the `^` should be removed

Reply:
Good points. Removed that commit from the PR.